l10nfr.properties

```diff
@@ -33,7 +33,19 @@ defaults.mavenArtifactId=sonar-l10n-fr-plugin
 9.9.0.changelogUrl=https://github.com/jycr/sonar-l10n-fr/releases/tag/9.9.0
 
 9.9.1.description=Fixes misleading translations and uses right apostrophe quotation mark
-9.9.1.sqVersions=[9.9,LATEST]
+9.9.1.sqVersions=[9.9,9.9.*]
 9.9.1.date=2023-05-31
 9.9.1.downloadUrl=https://github.com/jycr/sonar-l10n-fr/releases/download/9.9.1/sonar-l10n-fr-plugin-9.9.1.jar
 9.9.1.changelogUrl=https://github.com/jycr/sonar-l10n-fr/releases/tag/9.9.1
+
+9.9.2.description=Uses right quotation mark, fixes misleading translations, changes "demande de changement" by "Demande de Fusion" and "webhook" by "Notification Web"
+9.9.2.sqVersions=[9.9,LATEST]
```

Comment:
Only one version can be marked compatible with `LATEST`. This is about the Marketplace knowing which _single_ version of a plugin to recommend for which version of SonarQube. Set the end of this range to `9.9.*`